lib/iris/fileformats/_pyke_rules/fc_rules_cf.krb

```diff
@@ -1307,6 +1322,81 @@ fc_extras
 
         # Update the coordinate to CF-netCDF variable mapping.
         engine.provides['coordinates'].append((coord, cf_coord_var.cf_name))
+        
```

Comment:
This blank line contains spurious whitespace. Also, typically two blank lines would be left between one Python function and the next in this section of the file.